.ddev/config.yaml

```diff
@@ -0,0 +1,35 @@
+name: astrosite
```

Comment:
Recommend just leaving this out if possible. It will then take the name of the directory it's in.

```suggestion
```